stock_mts_mto_rule/__manifest__.py

```diff
@@ -12,5 +12,6 @@
     "application": False,
     "installable": True,
     "depends": ["stock"],
+    "maintainers": ["InsanivimusOmnes", "CetmixGitDrone"],
```

Comment:
@florian-dacosta Do you maybe want to add yourself here ?

Reply:
@rousseldenis I did not look in to it since a while, there is no need for me